site.yml

```diff
@@ -8,12 +8,14 @@
     - role: collectd-client
       tags: ['collectd-client','collectd']
       when: collectd.enabled|default('False')|bool
+  environment: "{{ env_vars|default({}) }}"
```

Comment:
I don't see how setting `environment` at a play level works, but I've only looked a little bit, do you have supporting docs on this working?